packages/cozy-client/src/queries/dsl.js

```diff
@@ -366,7 +366,7 @@ class QueryDefinition {
    * @param {string} id - The sharing doc id
    * @returns {QueryDefinition}  The QueryDefinition object.
    */
-  sharingId(id) {
+  setSharingId(id) {
```

Comment:
I don't really like this naming TBH. I understand that we technically set the sharingId to the QueryDefinition, but here the semantic should not be on the technical set, but rather from the querying perspective. As a developer, I would not understand doing `Q('io.cozy.files').setById(id)` to query a file by its id.
Likewise, I would prefer `sharingById` here

Reply:
I suggested `inSharing(id)`.

Reply:
Agree with `sharingById` !

Reply:
Oh @zatteo already called it

Reply:
Sorry I answered before seeing @taratatach comment...

I think `inSharing` is more explicit but like you said it is more "functional", and cozy-client API and naming is not functional so maybe we should stick with `sharingById`.